[2025-05-14T00:44:54.397+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_bank_reviews.run_clean_reviews manual__2025-05-13T23:44:36.782331+00:00 [queued]>
[2025-05-14T00:44:54.413+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_bank_reviews.run_clean_reviews manual__2025-05-13T23:44:36.782331+00:00 [queued]>
[2025-05-14T00:44:54.415+0100] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-14T00:44:54.449+0100] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): run_clean_reviews> on 2025-05-13 23:44:36.782331+00:00
[2025-05-14T00:44:54.462+0100] {standard_task_runner.py:60} INFO - Started process 18764 to run task
[2025-05-14T00:44:54.472+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'process_bank_reviews', 'run_clean_reviews', 'manual__2025-05-13T23:44:36.782331+00:00', '--job-id', '1890', '--raw', '--subdir', 'DAGS_FOLDER/post_processing.py', '--cfg-path', '/tmp/tmpg14rfk21']
[2025-05-14T00:44:54.481+0100] {standard_task_runner.py:88} INFO - Job 1890: Subtask run_clean_reviews
[2025-05-14T00:44:54.612+0100] {task_command.py:423} INFO - Running <TaskInstance: process_bank_reviews.run_clean_reviews manual__2025-05-13T23:44:36.782331+00:00 [running]> on host DESKTOP-RRUTCI2.
[2025-05-14T00:44:54.804+0100] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='process_bank_reviews' AIRFLOW_CTX_TASK_ID='run_clean_reviews' AIRFLOW_CTX_EXECUTION_DATE='2025-05-13T23:44:36.782331+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-13T23:44:36.782331+00:00'
[2025-05-14T00:44:54.810+0100] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-14T00:44:54.812+0100] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd ~/airflow/dbt_project && dbt run --select clean_bank_reviews']
[2025-05-14T00:44:54.831+0100] {subprocess.py:86} INFO - Output:
[2025-05-14T00:45:15.449+0100] {subprocess.py:93} INFO - [0m23:45:15  Running with dbt=1.8.7
[2025-05-14T00:45:18.628+0100] {subprocess.py:93} INFO - [0m23:45:18  Registered adapter: postgres=1.8.2
[2025-05-14T00:45:24.565+0100] {subprocess.py:93} INFO - [0m23:45:24  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-05-14T00:45:24.567+0100] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-05-14T00:45:24.568+0100] {subprocess.py:93} INFO - - models.dbt_project.forPy
[2025-05-14T00:45:25.442+0100] {subprocess.py:93} INFO - [0m23:45:25  Found 8 models, 1 seed, 19 data tests, 2 sources, 540 macros
[2025-05-14T00:45:25.448+0100] {subprocess.py:93} INFO - [0m23:45:25
[2025-05-14T00:45:27.673+0100] {subprocess.py:93} INFO - [0m23:45:27  Concurrency: 3 threads (target='dev')
[2025-05-14T00:45:27.675+0100] {subprocess.py:93} INFO - [0m23:45:27
[2025-05-14T00:45:27.679+0100] {subprocess.py:93} INFO - [0m23:45:27  1 of 1 START sql view model public.clean_bank_reviews .......................... [RUN]
[2025-05-14T00:45:27.897+0100] {subprocess.py:93} INFO - [0m23:45:27  1 of 1 OK created sql view model public.clean_bank_reviews ..................... [[32mCREATE VIEW[0m in 0.21s]
[2025-05-14T00:45:27.927+0100] {subprocess.py:93} INFO - [0m23:45:27
[2025-05-14T00:45:27.929+0100] {subprocess.py:93} INFO - [0m23:45:27  Finished running 1 view model in 0 hours 0 minutes and 2.48 seconds (2.48s).
[2025-05-14T00:45:27.990+0100] {subprocess.py:93} INFO - [0m23:45:27
[2025-05-14T00:45:27.992+0100] {subprocess.py:93} INFO - [0m23:45:27  [32mCompleted successfully[0m
[2025-05-14T00:45:27.994+0100] {subprocess.py:93} INFO - [0m23:45:27
[2025-05-14T00:45:27.995+0100] {subprocess.py:93} INFO - [0m23:45:27  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-05-14T00:45:28.882+0100] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-14T00:45:28.954+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=process_bank_reviews, task_id=run_clean_reviews, execution_date=20250513T234436, start_date=20250513T234454, end_date=20250513T234528
[2025-05-14T00:45:29.010+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-14T00:45:29.065+0100] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
